README.md

```diff
@@ -18,15 +18,15 @@ credentials "app.terraform.io" {
 }
 ```
 
-You can run `terraform format` at any time to format your code, or install the Terraform extension [for your editor](https://github.com/hashicorp/terraform-ls/blob/main/docs/USAGE.md).
+Run `make init` from this directory to install a [githook](https://github.com/DoSomething/infrastructure/blob/main/.githooks/pre-commit) to check formatting before you commit changes. You can run `make format` at any time to format your code, or install the Terraform extension [for your editor](https://github.com/hashicorp/terraform-ls/blob/main/docs/USAGE.md).
```

Comment:
I added directions for installing this repo's [githook](https://github.com/DoSomething/infrastructure/blob/main/.githooks/pre-commit) (that automatically checks formatting pre-commit). I also updated these instructions to suggest `make format`, which uses the `-recursive` flag to run this formatter on the whole repository (rather than just the current shallow directory).

Reply:
Noice! ✨